code/game/machinery/computer/camera.dm

```diff
@@ -96,7 +96,6 @@ var/list/obj/machinery/camera/cyborg_cams = list(
 	network = list(CAMERANET_THUNDER)
 	density = 0
 	circuit = null
-	layer = ABOVE_WINDOW_LAYER
```

Comment:
?

Reply:
That's not in the diff anymore so far as I can tell.

Reply:
Though it could be a result of splitting this off into a separate PR. I'll merge Bleeding-Edge into this branch, I guess.